File: praktikum/java/Human.java
public class Human{
    private String name;
    private char gender;

    public Human(){
        this.name ="";
        this.gender ='-';
    }

    public Human(String name ,char gender){
        this.name = name;
        this.gender = gender;
    }

    public String getname(){
        return this.name;
    }

    public void setname(String name){
        this.name = name;
    }

    public char getgender(){
        return this.gender;
    }

    public void setgender(char gender){
        this.gender = gender;
    }

    public void eat(){
        System.out.println(this.name + " is eating!");
    }

    public void sleep(){
        System.out.println(this.name + " is sleeping!");
    }
}

File: praktikum/c++/Human.cpp
#pragma once

#include <iostream>
#include <string>

using namespace std;

class Human{
    private:
        string name;
        char gender;
    public:
        Human(){
            this->name = "";
            this->gender = '-';
        }

        Human(string name,char gender){
            this->name = name;
            this->gender = gender;
        }

        string get_name(){
            return this->name;
        }

        void set_name(string name){
            this->name = name;
        }

        char get_gender(){
            return this->gender;
        }

        void set_gender(char gender){
            this->gender = gender;
        }

        void eat(){
            cout << this->name << " is eating!" << '\n';
        }

        void sleep(){
            cout << this->name << " is sleeping!" << '\n';
        }

        ~Human(){
            
        }

};